fe/fe-core/src/main/java/com/starrocks/backup/Repository.java

```diff
@@ -80,7 +80,7 @@
 public class Repository implements Writable {
     private static final Logger LOG = LogManager.getLogger(Repository.class);
 
-    public static final String PREFIX_REPO = "__palo_repository_";
+    public static final String PREFIX_REPO = "__starrocks_repository_";
```

Comment:
This value can not be changed because starrocks need to be compatible with old version' path

Reply:
sure